dev_guide/deployments.adoc

```diff
@@ -34,59 +34,60 @@ new deployment every time a Docker image tag changes:
 
 ====
 
-[source,json]
+[source.json]
```

Comment:
in other places you made the periods into commas, here you made a comma into a period.....?